### ISSUE TITLE
Performance issue Screenshots

### ISSUE THREAD

**Issue by ArmaWave** [Regalis11/Barotrauma#8510]
Bad performance after last updates

Bad performance no matter the place and huge fps drops when an enemy group spawns or when in combat

v0.16.7.0 Hotfix Windows

When idle fps were at an average of 30 and when in combat fps were at an average of 7, fps also drop when using turrets to an average of 15

![20220312102454_1](https://user-images.githubusercontent.com/93015760/158027217-b29d427b-09f6-447b-b4f8-2bc6e2619db9.jpg)
![20220312102735_1](https://user-images.githubusercontent.com/93015760/158027219-d711f791-0b1f-4c5d-b9c6-aaf4c2313820.jpg)
![20220312102746_1](https://user-images.githubusercontent.com/93015760/158027221-5364e290-2223-48de-a6ae-246f4d927a5b.jpg)
![20220312102806_1](https://user-images.githubusercontent.com/93015760/158027223-258b7dbb-e4fa-4edc-832d-8d8a52122f3c.jpg)
![20220312102818_1](https://user-images.githubusercontent.com/93015760/158027225-af28653f-9a23-4779-bf70-dc7621416f08.jpg)
![20220312102838_1](https://user-images.githubusercontent.com/93015760/158027226-091e5469-5e8c-40f7-af1e-22a0c6238b9c.jpg)
![20220312102858_1](https://user-images.githubusercontent.com/93015760/158027228-05521f15-d306-48b0-8d73-dcf9044cf795.jpg)
![20220312103101_1](https://user-images.githubusercontent.com/93015760/158027231-cfadfa2f-c8f3-4284-b8d0-866450b9f679.jpg)
![20220312094139_1](https://user-images.githubusercontent.com/93015760/158027233-bc964c5c-d95c-400b-9f6a-7d7ee999b468.jpg)
![20220312094805_1](https://user-images.githubusercontent.com/93015760/158027234-e33d8536-8fce-49b0-a5c9-f3875305e776.jpg)
![20220312100758_1](https://user-images.githubusercontent.com/93015760/158027235-e13de2ac-3c7c-4b1b-b9dd-fd3f7dee2efd.jpg)
![20220312100810_1](https://user-images.githubusercontent.com/93015760/158027236-a7ea5e5e-91d5-4c37-ae3a-3c9d4c46bf75.jpg)
![20220312101054_1](https://user-images.githubusercontent.com/93015760/158027237-3536c0ec-aa90-46fb-be02-3be769c03c65.jpg)
![20220312101056_1](https://user-images.githubusercontent.com/93015760/158027238-1a3e3e29-c8f3-4ea1-8d74-ad1d35b0735f.jpg)
![20220312101955_1](https://user-images.githubusercontent.com/93015760/158027239-242790fb-46a7-40eb-a894-8ea7a55a9780.jpg)
![20220312102231_1](https://user-images.githubusercontent.com/93015760/158027240-13f263c2-6611-42ae-ad52-b308524f762d.jpg)
![20220312102247_1](https://user-images.githubusercontent.com/93015760/158027242-ef87f6ab-6d1a-49d6-bbb9-5a2b9a77a97f.jpg)
![20220312102313_1](https://user-images.githubusercontent.com/93015760/158027243-05b0d3f9-870c-487c-8e25-f83ba374a9c1.jpg)
![20220312102407_1](https://user-images.githubusercontent.com/93015760/158027244-7da138b7-ddc3-4476-8dd3-3a78bb0b4f45.jpg)

**Comment by MateusJK15** [comment]
The last update messed the game's performance, there's even input lag when the fps falls below 20.

**Comment by ArmaWave** [comment]
yep input lag below 20 fps, half of the screenshots i had less than 20 fps, btw sent these screenshots cuz there was a discussion in steam and wanted to help a little cuz i wanna play

**Comment by PokkeTigrex** [comment]
In a server I'm in, pirate missions are a major culprit in performance, and only gets worse when fighting pirates as well as other creatures at the same time, game becomes uncontrollable frequently and basically unplayable with controls locking up from intense frame drops.

**Comment by BlackMoons** [comment]
Can confirm, On stable and pirate missions for my entire team where 5~15fps, more or less since the start of the mission on a campaign around 25~30% difficulty, dedicated server with 3.2ghz 6 core CPU and 1060GTX for my own PC. 

Oddly enough, tried some sandbox pirate missions, self hosted, they ran 40~60fps (No other players at the time), although I didn't go very far in them. 

In any case, I recommend testing for this bug using regular campaign missions in MP, not sandbox.
